module/epochs/epoch_lookup_test.go

```diff
@@ -7,13 +7,11 @@ import (
 	"time"
```

Comment:
Can you add a test which verifies that invalid `EpochExtension` doesn't update `cache[2]` and it's ignored? 
please cover all cases where the epoch extension can be invalid